api/HTMLInputElement.json

```diff
@@ -2254,9 +2254,15 @@
               },
               "chrome_android": "mirror",
               "edge": "mirror",
-              "firefox": {
-                "version_added": false
-              },
+              "firefox": [
+                {
+                  "version_added": "122"
+                },
+                {
+                  "version_added": "101",
+                  "version_removed": "112"
+                }
+              ],
               "firefox_android": "mirror",
```

Comment:
This should have the new data. The new data should also says 121 not 122 because it's been pulled up to the older version per the bug that you link